config/upgrade/v0.15.0/README.md

```diff
@@ -0,0 +1,116 @@
+# Upgrade script (optional) to upgrade to v0.15.0 of Eventing
+
+This directory contains a job that updates all of v1alpha1 based Brokers that
+are using Spec.ChannelTemplate to specify which channel they use. You are not
+able to create new ones specifying the ChannelTemplate, and you can either
+manually update them yourself, or run the tool provided here, which will do the
+following for any Broker that is using Spec.ChannelTemplate:
+
+1. Creates a ConfigMap in the same namespace as the Broker named:
+   `broker-upgrade-auto-gen-config-<brokername>` with the content from
+   Spec.ChannelTemplate.
+1. Set Broker Spec.Config to point to this ConfigMap
+1. Set Broker Spec.ChannelTemplate to nil
+
+To run the upgrade script:
+
+```shell
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.15.0/upgrade-to-v0.15.0.yaml
+```
+
+It will create a job called v0.15.0-upgrade in the knative-eventing namespace.
+If you installed to different namespace, you need to modify the upgrade.yaml
+appropriately. Also the job by default runs as `eventing-controller` service
+account, you can also modify that but the service account will need to have
+permissions to list `Namespace`s, list and patch `Broker`s, and create
+`ConfigMap`s.
+
+# Examples
+
+If for example you have an existing v1lpha1 Broker with the following Spec:
+
+```yaml
+spec:
+  channelTemplateSpec:
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: InMemoryChannel
+```
+
+The tool will create a ConfigMap that looks like so:
+
+```yaml
+apiVersion: v1
+data:
+  channelTemplateSpec: |2
```

Comment:
Is this 2 intentional?

Reply:
Yeah, it's because of showing the block there:
https://yaml.org/spec/1.2/spec.html#id2793979